ocs_ci/ocs/ocp.py

```diff
@@ -1106,3 +1105,95 @@ def verify_cluster_operator_status(cluster_operator):
     log.info(f'{cluster_operator} status is valid')
 
     return True
+
+
+def validate_cluster_version_status():
+    """
+    Verify OCP upgrade is completed, by checking 'oc get clusterversion'
+    status
+
+    Returns:
+        bool: False in case that one of condition flags is invalid:
+        Progressing (should be False), Failing(should be False)
+        or Available (should be True)
+
+    """
+    ocp = OCP(kind="clusterversion")
+    operator_data = ocp.get('-o json', out_yaml_format=False)
+    conditions = operator_data['items'][0].get('status').get('conditions')
+    for condition in conditions:
+        if condition['type'] == 'Progressing' and condition['status'] == 'True':
+            log.info('cluster version status is Progressing')
+            return False
+        if condition['type'] == 'Failing' and condition['status'] == 'True':
```

Comment:
use elif here and the same on line 1131. So if the condition will be met on line 1125 this code will not even try to check the other conditions.

Reply:
done